src/index.ts

```diff
@@ -48,7 +48,7 @@ export function getArbitrary<T extends HasArbitrary>(codec: T): fc.Arbitrary<t.T
   const type: HasArbitrary = codec as any;
   switch (type._tag) {
     case 'UnknownType':
-      return fc.anything();
+      return fc.anything() as any;
```

Comment:
Is it possible to use `unknown` here over `any`?

Reply:
It should be, but I kept `any` here for consistency with other lines below. Anyway, it does not make much difference in this case IMO